common/patches/rof2.cpp

```diff
@@ -5533,12 +5580,19 @@ namespace RoF2
 
 		//sprintf(hdr.unknown000, "06e0002Y1W00");
 
-		snprintf(hdr.unknown000, sizeof(hdr.unknown000), "%016d", item->ID);
+		if (packet_type == ItemPacketParcel) {
+			strn0cpy(
+				hdr.unknown000, fmt::format("{:03}PAR{:010}\0", inst->GetMerchantSlot(), item->ID).c_str(),
+				sizeof(hdr.unknown000));
+		}
+		else {
+			snprintf(hdr.unknown000, sizeof(hdr.unknown000), "%016d", item->ID);
+		}
```

Comment:
Default case can be declared above and remove the else

```cpp
snprintf(hdr.unknown000, sizeof(hdr.unknown000), "%016d", item->ID);
if (packet_type == ItemPacketParcel) {
	strn0cpy(
		hdr.unknown000, 
		fmt::format("{:03}PAR{:010}\0", inst->GetMerchantSlot(), item->ID).c_str(),
		sizeof(hdr.unknown000)
	);
}
```

Reply:
Resolved